maps/glloydstation/glloydstation_unit_testing.dm

```diff
@@ -101,9 +179,20 @@
 		/area/shuttle/infestation/i1/ship,
 		/area/shuttle/infestation/i2/station,
 		/area/shuttle/infestation/i2/ship,
+		/area/shuttle/train,
 		/area/shuttle/train/stop,
 		/area/shuttle/train/go,
 		/area/shuttle/naval1/event1,
 		/area/shuttle/naval1/event2,
-		/area/shuttle/naval1/event3
+		/area/shuttle/naval1/event3,
+		/area/shuttle/pirate1/centcom,
+		/area/shuttle/pirate1/station,
+		/area/shuttle/outpost/station,
+		/area/shuttle/outpost/jungle,
+		/area/syndicate_station/arrivals_dock,
+		/area/template_noop,
+		/area/overmap,
+		/area/turbolift,
+		/area/quartermaster/carpenter,
+		/area/quartermaster/hunter
```

Comment:
I meant the QM areas; if they aren't used, throw them out, let's not set a precedent for hoarding unused areas just in case.